app/views/Coach/Clients/index.js

```diff
@@ -80,18 +103,44 @@ class Clients extends Component {
             </div>
             </Tab>
             <Tab label="Oczekujący" >
-              <div>
-                <p>
-                TODO
-                </p>
-              </div>
+            <div style={styles.root}>
+              <GridList
+                cellHeight={350}
+                style={styles.gridList}
+              >
+                <Subheader>Oczekujący klienci</Subheader>
```

Comment:
Może spróbujemy dorzucić od razu oczekujących klientów? Coś czuję, że zrobi się to bardzo podobnie jak przy aktywnych